content/en/docs/pipelines/building-pipelines/overview.md

```diff
@@ -0,0 +1,83 @@
++++
+title = "Overview of Pipeline Development"
+description = "Getting started with pipeline development"
+weight = 10          
++++
+
+Kubeflow Pipelines lets you deploy and run portable, scalable machine learning
+(ML) workflows based on Docker containers. You can orchestrate your ML workflow
+as a pipeline using the Kubeflow Pipelines SDK or TensorFlow Extended (TFX).
+
+This document describes how to choose the best option for building your
+pipeline, and provides resources for getting started.
+
+## Selecting a pipelines SDK
+
+At a high level, the Kubeflow Pipelines SDK and TFX take a similar approach
+to defining pipelines. In both cases:
+
+*  A pipeline is a set of components and input parameters. 
+*  Component outputs are called artifacts. Components can accept either
```

Comment:
> Component outputs are called artifacts.

Ummm... As of today this is only true for TFX SDK because KFP also allows passing primitive-valued parameters between steps. See *Producing and consuming multiple arguments* in this [example](https://github.com/kubeflow/pipelines/blob/master/samples/tutorials/Data%20passing%20in%20python%20components.ipynb). So I'll recommend something along the line as the following:

Data passing between pipeline steps can be achieved by either component inputs/outputs, or wiring components with pipeline parameters.

Up to you.

Reply:
If I understand correctly, KFP component outputs can be artifacts or primitive-value parameters. Is that correct? From the users perspective, how are they different? Are they stored differently? Are there restrictions on a parameter that do not exist on an artifact? 

How about something like:

*  Component inputs can be set from pipeline parameters or the outputs of other components.

Reply:
> If I understand correctly, KFP component outputs can be artifacts or primitive-value parameters. Is that correct?

Yes this is true, but this paragraph is saying 'In both cases', which confused me a little bit. Perhaps I missed something here?